src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -92,7 +91,7 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
       avatar={handleAvatar()}
       isContinuous={item?.isContinuous ?? false}
       message={item?.message}
-      time={calculateTime(item?.updatedAt, true)}
+      time={item?.updatedAt}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `calculateTime` function was previously used to format the time. It seems like it has been removed and now the raw `updatedAt` value is being used. This might affect how the time is displayed in the chat messages.
```diff
-       time={item?.updatedAt}
+       time={calculateTime(item?.updatedAt, true)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->